bindings/go/runtime/registry.go

```diff
@@ -208,6 +208,10 @@ func (r *Scheme) RegisterWithAlias(prototype Typed, types ...Type) error {
 	rt := reflect.TypeOf(prototype).Elem()
 	defaultTyp, defaultExists := r.defaults.GetRight(rt)
 	if !defaultExists {
```

Comment:
So, if no types were passed and a default does exist, we will return nil and do nothing. That seems odd